### PR TITLE
fix interactive for personachat seq2seq

### LINKED ISSUE #1430: error in running seq2seq_withprofile_interactive.py

### PR DESCRIPTION
**Pull request by alexholdenmiller**
Fixes #1430. I think got a transient error but I got these errors when I ran that script myself

### CODE REVIEW

File: projects/personachat/persona_seq2seq.py [hunk 1]
```diff
@@ -197,12 +200,12 @@ def __init__(self, opt, shared=None):
             self.NULL_IDX = self.dict.txt2vec(self.dict.null_token)[0]
 
             # reorder dictionary tokens
-            self.dict.ind2tok[1] = '__END__'
-            self.dict.tok2ind['__END__'] = 1
-            self.dict.ind2tok[2] = '__UNK__'
-            self.dict.tok2ind['__UNK__'] = 2
-            self.dict.ind2tok[3] = '__START__'
-            self.dict.tok2ind['__START__'] = 3
+            # self.dict.ind2tok[1] = '__END__'
```
Comment (stephenroller) [inline projects/personachat/persona_seq2seq.py]: Maybe we should just delete the whole block?

**Comment by stephenroller** [comment]
Yeah pretty sure that bug was just a failed download.

Fixes #1430

**Comment by stephenroller** [comment]
Travis is just complaining about lint on this legacy code.

**Comment by jaseweston** [comment]
how is this still here?

**Comment by stephenroller** [comment]
Drop the newly commented lines and test

**Comment by alexholdenmiller** [comment]
talking to the model, it looks like it's actually having the start and end indices swapped so it needs some touchup

**Comment by emilydinan** [comment]
closing this since we deprecated this model with #1581

**Pull request closed without merging**